source/documentation/using_ci/index.md

```diff
@@ -0,0 +1,13 @@
+# Configuring a custom continuous integration (CI) system
+
+Setting up a CI service lets you make sure your code changes are delivering new features without introducing new bugs or breaking existing functionality. This helps you to iterate code faster and with greater confidence, a huge benefit once the codebase becomes larger and more complex. 
+ 
+For advanced projects that need high levels of control over configuration, a tool like [Jenkins](https://jenkins.io/) is often used. For simpler projects, or where a team needs to manage deployments of prototypes from their github repositories rather than local code, cloud-based services such as [Travis CI](https://travis-ci.com/) or [CircleCI](https://circleci.com/) are common. Instructions to configure both Jenkins and Travis to run on GOV.UK PaaS are below.
```

Comment:
What I don't want is for someone inexperienced to choose Jenkins because they think they might need "high levels of control" or their app is "more advanced than a prototype".

I think it's enough to say what they could use - probably in the order of "Travis CI, Circle CI, Jenkins, etc" - and that we have instructions for Travis and Jenkins.